app/views/topics/new.html.erb

```diff
@@ -0,0 +1,22 @@
+<h2>Topics</h2>
+<p>Add a new topic</p>
+
+<div>
+  <%= form_for @topic do |f| %>
+    <%= f.label :name %>
+    <%= f.text_field :name, id: :name %>
+    <%= f.submit %>
+  <% end %>
+</div>
+
+
+<% if @topic.errors.any? %>
```

Comment:
It's more usual to put the errors towards the top of the form, so they are the first thing the user sees when the page loads.